### PR TITLE
Add installation hints from TU Repo to README.md

### PR DESCRIPTION
**Pull request by fabiankrieg**

### CODE REVIEW

**Comment by SebastianSemper** [comment]
Can someone please verify, that the added instructions are still up to date?

**Comment by ChristophWWagner** [comment]
Looks quite so. Except that you're now able to also build docs and benchmarks. Would suggest rewriting the instructions though as probably most users would just want to get started with using, not debugging

**Comment by fabiankrieg** [comment]
I would guess so, too. But for me it was rather helpfull, as I just 
wanted a local installation. So including the Non-Pip way and 
dependencies somewhere at the high level docu does not seem bad to me. 
One could cluster the installation section into: 1.getting started -> 
Pip 2. Build from source !?


Am 14.11.2017 um 20:36 schrieb Christoph Wagner:

File: README.md [hunk 1]
```diff
@@ -90,13 +90,36 @@ greatly improving installation time and effort, especially under windows.
...
     https://github.com/EMS-TU-Ilmenau/fastmat/archive/master.zip
 - unpack its contents and navigate to the project root directory
-- make sure you have installed all listed dependencies with at least the
-  specified version
-- type `make install` to install fastmat on your computer
-    * If you intend to install the package locally for your user
-      type `make install MODE=--user` instead.
-    * If you only would like to compile the package to use it from this local
-      directory without installing it, type `make compile`
+#### LINUX
```
Comment (ChristophWWagner) [inline README.md]: These sections should be cleaned up. As this is a very very old description, it does not feature pip at all, which is currently the preferred isntallation method. I would suggest restructuring the whole installation chapter to the following key points:

# Intallation

## using pip
 --> pip install fastmat

## manually
  put descriptions on where to get the source from, what to do with it all those descriptions on make file usage here (also point to the relevant wiki page of this repo)

## Requirements for Building on Windows
  as no compiler exists natively ... bla bla
  our recommendations: Intel Python, ... bla bla

**Pull request merged**